seatunnel-core/seatunnel-core-base/src/main/java/io/github/interestinglab/waterdrop/config/ConfigBuilder.java

```diff
@@ -154,23 +153,16 @@ public void checkConfig() {
     }
 
     public <T extends Plugin> List<T> createPlugins(PluginType type) {
-
```

Comment:
Hi, these changes are not related to the patch purpose, it's better to keep it.

Reply:
done.